views/includes/head.pug

```diff
@@ -1,10 +1,27 @@
 head
   title= title
   meta(charset='utf-8')
+  meta(name='viewport' content='width=device-width, initial-scale=1')
+  link(rel='shortcut icon' type='image/png' href='/public/favicon.png')
+  link(rel='apple-touch-icon' href='/public/favicon.png')
   link(
     rel='stylesheet',
-    href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css',
-    integrity='sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u',
+    href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css',
+    integrity='sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb',
+    crossorigin='anonymous'
+  )
+  link(rel='stylesheet', href='public/css/style.css')
+  script(
+    src='https://code.jquery.com/jquery-3.2.1.slim.min.js'
```

Comment:
Do we use `jQuery` and `popper`?

Reply:
It's used on mobile version for the navbar toggler.

Reply:
For the Bootstrap. I don't want to implement it if it's part of bootstrap.